integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/hooks/ScenarioHooks.java

```diff
@@ -55,13 +55,29 @@ public class ScenarioHooks {
    * <p>Order 1000 ensures this will be run as one of the first hooks before the scenario.
    */
   @Before(order = 1000)
-  public void beforeScenario() {
+  public void beforeScenario() throws IOException {
+    this.loadConfiguration();
     this.deviceSimulatorSteps.clearDlmsAttributeValues();
     this.dlmsDatabase.prepareDatabaseForScenario();
     this.wsSmartMeteringNotificationDatabase.prepareDatabaseForScenario();
     this.prepareServiceEndpoint();
   }
 
+  private void loadConfiguration() throws IOException {
```

Comment:
Dit had ik verwacht. Kun je deze niet herbruikbaar maken in een aparte class file. Property file inladen bij laden object en dan een methode waarmee je properties kan ophalen. Dan kun je die ook gebruiken in de class hierboven.